Svg/SvgDocument.cs

```diff
@@ -647,6 +647,13 @@ public RectangleF CalculateDocumentBounds()
         {
             RectangleF documentSize = null;
 
+            if (Width != SvgUnit.None && Height != SvgUnit.None)
```

Comment:
please do not do that.
This is "DrawAllContent" - so it should do just that. Otherwise we will have bugs in our Sketch Editor and so on.

Please add another method "DrawDocument" that does what _you_ need for your scenario!